frontend/src/components/DataCard/SuggestedLabels.jsx

```diff
@@ -1,15 +1,24 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 
-export default function SuggestedLabels({ card, labels }) {
-    if (labels.length <= 5) {
+export default function SuggestedLabels({ card, labels, onSelectLabel }) {
+    const [suggestions, setSuggestions] = useState();
+
+    useEffect(() => {
+        fetch(`/api/comparisons/${card.text.project}?text=${card.text.text}`)
```

Comment:
Ok so I'm ok with this fetch going here, but I do think we'll want to add some error handling. If you look in any of the action files, you'll see some examples.